templates/implnet_ops_SOURCEVAL.py

```diff
@@ -293,177 +260,145 @@ def _create_service(
         networks=container_context.networks
         if len(container_context.networks)
         else None,
-        restart_policy=restart_policy,
-        mode=serivce_mode,
+        restart_policy= RestartPolicy(condition="none"),
+        mode=ServiceMode("replicated-job", concurrency=1, replicas=1),
         workdir=workingdir,
-        configs=configs,
```

Comment:
Don't like the pattern of defining a variable just to use it in func arg if it is very minor